packages/core/src/components/datetime-picker/datetime-picker.scss

```diff
@@ -35,23 +32,29 @@
 
   @media (min-width: 576px) {
     min-width: max-content;
+    --btn-width: min-content;
 
-    .btn-select-date {
-      left: unset !important;
+    .btn-select-date-container {
+      position: absolute;
```

Comment:
Do we really need a absolute position here? Looks like the data container can also be placed using `relative` position.